api/views.py

```diff
@@ -67,7 +67,7 @@ class OrganizationViewSet(PublicationMixin, viewsets.ReadOnlyModelViewSet):
 class ProjectViewSet(PublicationMixin, viewsets.ReadOnlyModelViewSet):
     queryset = Project.objects.distinct().select_related(
         'infrastructure_type',
-    ).all()
+    ).filter(published=True)
```

Comment:
This overrides the functionality of the ``PublicationMixin``, which will filter projects based on the ``published`` boolean depending on whether the user is logged in. Is that intentional?

Reply:
Once I tested this as a normal user and a logged in user. I saw the behavior I was expecting. The override was unnecessary. Reverted.